src/routes.js

```diff
@@ -9,7 +9,7 @@ import history from './history';
 import Applications from './sections/applications';
 import Charts from './sections/charts';
 import Components from './sections/components';
-import Dashboard from './sections/dashboard';
+import DashboardPage from './sections/dashboard';
```

Comment:
why this change?
it is better?
I think we should name it only Dashboard, also all other don't have the Page suffix and don't need it since all are pages.